contracts/ConverterRamp.sol

```diff
@@ -96,29 +115,33 @@ contract ConverterRamp is Ownable {
             _callbackData
         );
         
-        // Transfer loan to msg.sender
+        /// transfer loan to msg.sender
         DebtEngine(_debtEngineAddress).transferFrom(address(this), msg.sender, uint256(_requestId));
 
-        require(token.safeApprove(_loanManagerAddress, 0), 'Error removing approve');
-        require(token.balanceOf(address(this)) == 0, 'The contract balance should be zero');
+        require(token.safeApprove(_loanManagerAddress, 0), 'error removing approve');
+        require(token.balanceOf(address(this)) == 0, 'the contract balance should be zero');
 
     }
 
+    /// @notice get the cost, in wei, of making a payment of the specified value 
```

Comment:
of making a payment using the value specified by the payment token of this contract.